pkg/server/invoke.go

```diff
@@ -35,9 +36,32 @@ func (s *DriverServiceServer) Invoke(ctx context.Context, req *pb.InvokeRequest)
 		return nil, status.Errorf(codes.InvalidArgument, "Error making client: %v", err)
 	}
 
-	action, ok := actions.Actions[req.GetMethod()]
+	method := req.GetMethod()
+
+	vmid, err := one.GetVMIDFromData(client, req.Instance)
+	if err != nil {
+		s.log.Sugar().Errorf("VM id obtaining failed for id=%s", req.Instance.Uuid)
```

Comment:
1. Wrong Log format, standard zap.Fields functionality must be used, log level must be WARN